### PR TITLE
add 2 game stats methods

### PR DESCRIPTION
**Pull request by z-fitch**
This is a revised PR for My game stats methods. Only the methods the tests will come later

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -57,13 +57,39 @@ def percentage_visitor_wins
     (away_win?.to_f / @games.size).round(2)
   end
 
+  def percentage_ties
```
Comment (MWoodshop) [inline lib/stat_tracker.rb:60]: This works perfectly but could potentially be shortened with something like this:
`def percentage_ties`
`tie_games = @game_teams.count { |game| game.game_result.include?("TIE") }`
`  (tie_games.to_f / @game_teams.length.to_f).round(2)`
`end`

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -57,13 +57,39 @@ def percentage_visitor_wins
...
+      @games.find_all do |game|
+        ids = game.season_id
+        only_ids << ids
+      end
+    uniq_ids = only_ids.uniq
+    count_games = []
+    final_count = Hash.new
+      uniq_ids.each do |id|
+        count_games = @games.find_all {|game|game.season_id.include?(id)}
+        final_count[id] = count_games.length
```
Comment (MWoodshop) [inline lib/stat_tracker.rb:80]: I believe you can also use `count` here instead of `length` as you do below. This still works correctly though.

**Review by MWoodshop — APPROVED** [review]
Added some comments / suggestions but overall code works as intended.

**Review by awiley33 — APPROVED** [review]
I made some technical comments on the prior version of this PR - this one also looks good!

**Pull request merged**